lib/index.js

```diff
@@ -32,7 +31,6 @@ api.lint = standalone;
 api.rules = requiredRules;
 api.formatters = formatters;
 api.createPlugin = createPlugin;
-api.createRuleTester = createRuleTester;
```

Comment:
We need to put this back, as we discussed in https://github.com/stylelint/stylelint/issues/4267. We'll only deprecate this API in the following release.